spt/OrangeBox/spt-serverplugin.cpp

```diff
@@ -904,4 +928,77 @@ CON_COMMAND(y_spt_find_seam_shot, "y_spt_find_seam_shot [<pitch1> <yaw1> <pitch2
 
 	Msg("Could not find a seam shot. Best guess: setang %.8f %.8f 0\n", test.x, test.y);
 }
+
+
+CON_COMMAND(tas_script_load, "Loads and executes an .stas script. Usage: tas_load_script [script]")
+{
+#if defined( OE )
+	if (!engine)
+		return;
+
+	ArgsWrapper args(engine.get());
+#endif
+
+	if (args.ArgC() > 1)
+	{
+		scripts::g_TASReader.ExecuteScript(args.Arg(1));
+	}
+	else
+		Msg("Loads and executes an .stas script. Usage: tas_load_script [script]\n");
+
+
+}
+
+CON_COMMAND(tas_script_search, "Starts a variable search for an .stas script. Usage: tas_load_search [script]")
+{
+#if defined( OE )
+	if (!engine)
+		return;
+
+	ArgsWrapper args(engine.get());
+#endif
+
+	if (args.ArgC() > 1)
+	{
+		scripts::g_TASReader.StartSearch(args.Arg(1));
+	}
+	else
+		Msg("Starts a variable search for an .stas script. Usage: tas_load_search [script]\n");
+
+
+}
+
+CON_COMMAND(tas_script_result_success, "Returns a successful result in a variable search.")
+{
+	scripts::g_TASReader.SearchResult(scripts::SearchResult::Success);
+}
+
+CON_COMMAND(tas_script_result_fail, "Returns an unsuccessful result in a variable search.")
+{
+	scripts::g_TASReader.SearchResult(scripts::SearchResult::Fail);
+}
+
+CON_COMMAND(_y_spt_findangle, "Finds yaw/pitch angle to position from player's current position.")
```

Comment:
"Finds the yaw/pitch angle required to look at the given position from player's current position."?

Reply:
This was mostly for portal, it's for finding the angle you need to look at when you need to shoot a portal in a certain position.

Reply:
Yeah then my description should be correct. Do you think it's clearer?

Reply:
changed to yours